plugins/node/opentelemetry-instrumentation-pino/package.json

```diff
@@ -45,7 +45,6 @@
   },
   "devDependencies": {
     "@opentelemetry/api": "^1.3.0",
-    "@opentelemetry/context-async-hooks": "^1.8.0",
```

Comment:
Reviewer note: This devDep was being used for:

```js
context.setGlobalContextManager(new AsyncHooksContextManager());
```

which isn't necessary in the tests because the only tracing being done is "create a span context and then immediately/synchronously log in that context". There is no need for tracking context through any async actions.